content/en/blog/2022-07-11-gitopsdays-vscode-extension-demo/index.md

```diff
@@ -0,0 +1,79 @@
+---
+author: 'kingdonb'
+date: 2022-07-11 14:30:00+00:00
+title: 'GitOps Days - VSCode Extension Demo'
+description: 'If you want GitOps in your IDE, check out this post and video from Kingdon who presented the GitOps VSCode Extension at the recent GitOps Days event.'
+url: /blog/2022/07/gitopsdays-vscode-extension-demo/
+tags: [event]
+resources:
+- src: "**.{jpg,png}"
+  title: "Image #:counter"
+---
+
+Helping to close out GitOps Days 2022, Kingdon Barrett maintainer of the
+Weaveworks [GitOps Tools extension for
+VSCode](https://github.com/weaveworks/vscode-gitops-tools) presents the
+new Flux extension. Kingdon, OSS Engineer at Weaveworks and Flux Maintainer,
+shows the new extension and how it helps minimize context switching,
+keeping you in your editor where you can be most productive!
```

Comment:
What if we rephrase to this so the tense is consistent:

Helping to close out GitOps Days 2022, Kingdon Barrett, OSS Engineer at Weavworks, Flux maintainer, and maintainer of the Weaveworks [GitOps Tools extension for
VSCode](https://github.com/weaveworks/vscode-gitops-tools) presented the new Flux extension. Kingdon showed the new extension and how it helps minimize context switching, keeping you in your editor where you can be most productive! Below is a recap and link to the video.